Makefile

```diff
@@ -61,7 +61,7 @@ gen:
 
 nag:
 	@echo "=== cfn_nag ==="
-	@gem list -i cfn-nag --silent || sudo gem install cfn-nag
+	@gem list -i cfn-nag --silent || gem install cfn-nag || sudo gem install cfn-nag
```

Comment:
Can you add the `--user-install` flag to gem install - that should always work and not require sudo.

Also, if it fails to install that way I'd prefer be notified and take action myself instead of assuming install with sudo.

Reply:
Done.